README.md

```diff
@@ -77,59 +86,124 @@ If you just want to run the system, see "Run the Code" below.
 1. Setup the build-environment.
    Note that this script is just a convenience to install system-wide dependencies we expect.
    As a result, it uses the system package manager, requires `sudo`, and should only be run **once**.
+
+   ```console
+   sudo ./scripts/install-build-tools.sh
+   ```
+
+   Note: Running Homebrew as root on mac via shell script is not supported, so run without sudo and when prompted, enter the root password.
+
    ```console
-   # ./scripts/install-build-tools.sh
+   ./scripts/install-build-tools.sh
```

Comment:
The purpose of using # and $ are to indicate when a root shell is in use and when a user shell is in use and we do this throughout our docs -- so I would keep these in (though I acknowledge they make copy paste annoying)